### DISCUSSION TITLE
What does a keyword do?

### DISCUSSION THREAD

**Discussion by quovadis-7** [cue-lang/cue#1345] [Q&A] [answered]
In this section, it says that CUE reserves all identifiers starting with __(double underscores) as keywords.
In which case I could use an identifier starting with __, can  you show me an example?

**Comment by verdverm** [comment, marked as answer]
[https://github.com/cue-lang/cue/search?q=__](https://github.com/cue-lang/cue/search?q=__) looks to have some breadcrumbs. As I understand it, they are largely internal and you shouldn't see them as a user.

You can use quotes to have labels which contain special chars

```cue
"#def": "not really a definition"
"_hid": "not really hidden"
"__foo": "not really a reserved identifier"
```

https://cuelang.org/play/?id=ysVprjTceqV#cue@export@json

**Reply by myitcv** [reply]
Thanks @verdverm. 

@quovadis1234 - I converted your issue to be a discussion as it's a better fit.
